attributes/default.rb

```diff
@@ -85,6 +85,9 @@
 default['rundeck_server']['rundeck-config.framework']['framework.ssh.user']         = 'rundeck'
 default['rundeck_server']['rundeck-config.framework']['framework.ssh.timeout']      = 0
 
+# realm.properties users
+default['rundeck_server']['realm.properties']['admin'] = "admin,user,admin,architect,deploy,build"
```

Comment:
I think the defaults should be (according to the [docs](http://rundeck.org/docs/administration/authenticating-users.html#propertyfileloginmodule)):
```
default['rundeck_server']['realm.properties']['admin'] = "admin,user,admin"
default['rundeck_server']['realm.properties']['user'] = "user,user"
```